examples/canvas.rs

```diff
@@ -160,8 +162,12 @@ impl App {
     }
 
     fn boxes_canvas(&self, area: Rect) -> impl Widget {
-        let (left, right, bottom, top) =
-            (0.0, area.width as f64, 0.0, area.height as f64 * 2.0 - 4.0);
+        let (left, right, bottom, top) = (
+            0.0,
+            f64::from(area.width),
+            0.0,
+            f64::from(area.height).mul_add(2.0, -4.0),
+        );
```

Comment:
```suggestion
        let left = 0.0;
        let right = f64::from(area.width);
        let bottom = 0.0;
        let top = f64::from(area.height).mul_add(2.0, -4.0);
```

Reply:
After seeing the suggestion this change is really obvious :D